rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,22 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+references:
+    - https://attack.mitre.org/techniques/T1553/004/
```

Comment:
could you please delete a link to the ATT&CK (as the mapping is already present in tags) and put there a link to the [relevant Atomic Red Team test](https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1553.004/T1553.004.md) instead?

Reply:
Hello @omergunal! could you please add a link to the relevant Atomic Red Team test to the `references`?

Reply:
Thank you!